component-catalog/src/Examples/Table.elm

```diff
@@ -135,6 +134,80 @@ example =
 
                 ( False, True ) ->
                     Table.viewLoadingWithoutHeader [] columns
+            , Heading.h2
+                [ Heading.plaintext "Using placeholderColumn for consistent widths tables"
```

Comment:
"for consistent column alignment", maybe?

I think it will be hard to condense all the information of the use case in a header, though. How about adapting the very nice docs you added for the new functions and adding it as a little intro blurb right below the header? Something like:

> By default tables will distribute any remaining space across columns. You can change this behavior to have fixed column widths and leave any remaining blank space to the right by using empty placeholder columns with `placeholderColumn`. This is particularly useful when you need to align similar columns across multiple tables.